draft/draft-ietf-tcpm-yang-tcp.xml

```diff
@@ -259,7 +280,7 @@
       </section>
 
       <section title="Tree Diagram">
-        <t>This section provides a abridged tree diagram for the YANG module
+        <t>This section provides a tree diagram for the YANG module
```

Comment:
Truth be told, it is an abridged tree diagram. We decided to include the full tree diagram in the appendix.

Reply:
OK, I have just reverted to the original text. This was a comment from Med.